Controller/Adminhtml/Integration/CreateStore.php

```diff
@@ -146,9 +150,7 @@ public function generateSearchEngineData($websiteID)
                             [
                                 'type' => 'magento2',
                                 'options' => [
-                                    'url' => $this->urlInterface->getBaseUrl() . 'rest/' . $store->getCode() . '/V1/',
-                                    'token' => $integrationToken,
-                                    'website_id' => $store->getWebsiteId(),
+                                    'url' => $this->urlInterface->getBaseUrl() . 'rest/' . $store->getCode() . '/V1/'
```

Comment:
We need to make sure that the website_id is being obtained in doofeeds from the store in order to obtain the corresponding products from it, if we make this change.

Reply:
We talked over it, and this is getting handled in [this](https://github.com/doofinder/doomanager/pull/2832) PR